Source/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -6,7 +6,8 @@
 //  Copyright © 2015 Realm. All rights reserved.
 //
 
-import SwiftLintFramework
+@testable import SwiftLintFramework
```

Comment:
I think it's fine to make `isEqualTo` part of the public API, therefore unnecessary to import this as `@testable`